### PR TITLE
ipvsadm: update version to 1.30

### PR DESCRIPTION
**Pull request by feckert**
Maintainer: me
Compile tested: lantiq_xrx200 and x86_64, APU3, OpenWrt master
Run tested: x86_64, APU3, Openwrt master, only execute command

Description:
Update version to 1.30
Replace libnl-tiny with libnl-genl

### CODE REVIEW

File: net/ipvsadm/Makefile [hunk 1]
```diff
@@ -9,15 +9,15 @@
...
 PKG_NAME:=ipvsadm
-PKG_VERSION:=1.29
+PKG_VERSION:=1.30
 PKG_MAINTAINER:=Mauro Mozzarelli <mauro@ezplanet.org>, \
 		Florian Eckert <fe@dev.tdt.de>
 PKG_LICENSE:=GPL-2.0+
 PKG_RELEASE:=1
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.xz
 PKG_SOURCE_URL:=https://www.kernel.org/pub/linux/utils/kernel/ipvsadm/
```
Comment (neheb) [inline net/ipvsadm/Makefile]: This can be replaced with

@KERNEL/linux/utils/kernel/ipvsadm

File: net/ipvsadm/Makefile [hunk 2]
```diff
@@ -9,15 +9,15 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=ipvsadm
-PKG_VERSION:=1.29
+PKG_VERSION:=1.30
 PKG_MAINTAINER:=Mauro Mozzarelli <mauro@ezplanet.org>, \
 		Florian Eckert <fe@dev.tdt.de>
 PKG_LICENSE:=GPL-2.0+
```
Comment (neheb) [inline net/ipvsadm/Makefile]: This should be GPL-2.0-or-later

File: net/ipvsadm/Makefile (hunk 1 above)
Comment (feckert) [reply net/ipvsadm/Makefile]: Thanks for the hint.
I have update my PR

File: net/ipvsadm/Makefile (hunk 2 above)
Comment (feckert) [reply net/ipvsadm/Makefile]: Thanks for the hint.
I have update my PR

**Pull request merged**